### DISCUSSION TITLE
What happened with Chibaku

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#453] [Q&A] [answered]
the /ct command, may i know why is it removed?

**Comment by winstxnhdw** [comment, marked as answer]
Replaced with `spin`. Always wanted to get rid of `ct` ever since the updates broke building outside the ship.

**Reply by Dexteriority** [reply]
How does it work again? i typed it but the cupboard just stay in place

**Reply by winstxnhdw** [reply]
I made a mistake. Try again.

**Reply by Dexteriority** [reply]
Still the same? I dont really know what is happening XD

**Reply by winstxnhdw** [reply]
I'll fix it another time.

**Reply by winstxnhdw** [reply]
Nvm, the bug is obvious. I fixed it.
